tests/test_coverage_boost.py

```
@@ -1,499 +1,292 @@
+
+        # Test line 442-452 indirectly by executing the callback logic
+        class Event:
+            args = ["root", "root/sub"]
+
+        # We simulate what the callback would do
+        old_expanded = set(m._props.get('expanded', []))
+        new_expanded = set(Event.args)
+        added = new_expanded - old_expanded
+
+        if added:
+            target = list(added)[0]
```

Ruff (0.15.12)

[warning] Prefer `next(iter(added))` over single element slice

Replace with `next(iter(added))`

(RUF015)